packages/renderer/src/App.svelte

```diff
@@ -75,7 +75,7 @@ window.events?.receive('display-help', () => {
         <div
           class="flex order-none title-font font-medium items-center text-white align-middle justify-center mb-4 md:mb-0">
           <Logo />
-          <span class="ml-3 text-xl block text-gray-300">Podman Desktop</span>
+          <span class="cursor-default ml-3 text-xl block text-gray-300">Podman Desktop</span>
```

Comment:
```suggestion
          <span class="select-none ml-3 text-xl block text-gray-300">Podman Desktop</span>
```

I think you should avoid text selection instead of just using a different cursor
Else even if we have a pointer cursor, moving rapidly on the left or the right the cursor is still selecting text
![image](https://user-images.githubusercontent.com/436777/215089885-b210fe1f-cd27-493e-9fbf-801630e0e0c3.png)